modules/data.js

```diff
@@ -22,7 +22,7 @@ export default class Books {
         }, 3000);
       }
       if (writer === '') {
```

Comment:
- [ ] The whole point of this milestone is to use `modules` as a way of learning how to do modular programming. Currently you only have 2 modules relevant to this project, `data.js` and `time.js`. Consider moving the `selectItems()` to a separate `.js` file. Also move the `removeItem = () => {}` function out of the `selectItems()` function into its separate `.js` file. Then you can import it back into the `selectItems()` function, and then import the `selectItems()` function back into `data.js` for a more modular approach.

![image](https://github.com/rmiyoyo/Awesome-Books-v2-With-ES6/assets/81932478/36492d00-ee69-4876-b3f9-20b03c64d272)